docs/specification.rst

```diff
@@ -128,7 +128,7 @@ Formal syntax specification of a line of a FASM file
 Canonicalization
 ++++++++++++++++
 
-If two FASM files have been canonicalized and are identical, then they enable an identical set of features. The canonical FASM file is also equivalent to the FASM file that would be generated by taking the output bitstream and converting it back into a FASM file.
+If two FASM files have been canonicalized, then they enable an identical set of features. The canonical FASM file is also equivalent to the FASM file that would be generated by taking the output bitstream and converting it back into a FASM file.
```

Comment:
This edit is wrong?  Two canonical FASM files that are also equal enable the same features.  You've dropped the `and equal` bit, which is important

Reply:
Right, restored the previous version.